src/styles/vantagem.module.css

```
@@ -1,13 +1,19 @@
 .vantagem {
-    background-color: #E6D9B1;
     border-bottom: solid 1px #A68444;
+}
+
+.vantagem_header {
```

Já que `.vantagem_header ` tem `display: flex` é mais fácil controlar o espaçamento entre os item usando a propriedade `gap`
```suggestion
.vantagem_header {
    gap: 4rem;
```